server/src/main/java/org/opensearch/identity/noop/NoopAccessTokenManager.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.identity.noop;
+
+import org.opensearch.identity.AccessToken;
+import org.opensearch.identity.AccessTokenManager;
+
+/**
+ * Implementation of access token manager that does not enforce authentication
+ *
+ * This class and related classes in this package will not return nulls or fail permissions checks
+ *
+ * @opensearch.internal
+ */
+public class NoopAccessTokenManager implements AccessTokenManager {
+
+    @Override
+    public void expireAllTokens() {
+        // Tokens cannot be expired
```

Comment:
Should we add one more line explaining "why"?

Reply:
`NoopAccessTokenManager` and all members of the `Noop` package does not allow for any auth failures/rejections of any kind (at least at this stage).  We can certainly revisit as implementations start coming into place.